src/OSPSuite.Core/Domain/ISimulation.cs

```diff
@@ -1,38 +1,17 @@
 ﻿using System.Collections.Generic;
 using OSPSuite.Core.Chart;
-using OSPSuite.Core.Domain.Builder;
 
 namespace OSPSuite.Core.Domain
 {
-   public interface ISimulation : IObjectBase, ILazyLoadable, IAnalysable, IWithCreationMetaData, IWithModel, IUsesObservedData
+   public interface ISimulation : ILazyLoadable, IAnalysable, IUsesObservedData, IModelCoreSimulation
```

Comment:
I made ISimulation inherit from IModelCoreSimulation. 
Simplifies a lot of the migration